chapters/equations.tex

```diff
@@ -771,14 +771,14 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \lstinline!start!-values of variables having \lstinline!fixed = false! can be used as initial guesses, in case iterative solvers are used in the initialization phase.
 
 \begin{nonnormative}
-In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value of the \lstinline!start!-attribute as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
+In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
 \end{nonnormative}
 
-If a parameter has a modifier for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the modifier for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
+If a parameter has a value for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the value for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
 In this case a diagnostic message is recommended in a simulation model.
 
 \begin{nonnormative}
-This is used in libraries to give non-zero defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
+This is used in libraries to give appropriate defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
```

Comment:
If the defaults would have been "appropriate", it would have been more natural to provide them as value modifiers instead.  Can we find a formulation that doesn't suggest that this is good modeling style?
```suggestion
This is used in libraries to give dummy defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
```

Reply:
Dummy is too far off for me, as it indicates it isn't really a default. After using a thesaurus I have the alternatives:

- Acceptable
- Permissible
- Tolerable
- Adequate

I believe the first two are acceptable.

Reply:
Maybe what they are is simply _defaluts_, but I see a risk of causing confusion as setting a value for the parameter is probably also considered giving a default:
```
  parameter SI.Resistance R = 1; /* Here, the 1 is also a default in one sense */
```

Would it be better to call it a _fallback value_?  (Also, these parameters obviously don't need to be set, as there is a fallback.)
```suggestion
This is used in libraries to provide a fallback value so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that should be set properly.
```

Reply:
I changed to _acceptable_.